### PR TITLE
new icon: googlecloud (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by EnisMulic**
Taken from #170

### CODE REVIEW

**Comment by amacado** [comment]
Hi @EnisMulic! Nice to see you have decided to pick up some old pull requests and fix them. We are currently investigating some issues regarding our `bot:peek` command (https://github.com/devicons/devicon/issues/414) so it can take some time until the review. Sorry for that.

**Comment by EnisMulic** [comment]
That's fine, I still have a few icons from this PR to fix up.

File: icons/googlecloud/googlecloud-plain.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```
Comment (amacado) [inline icons/googlecloud/googlecloud-plain.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 
	 viewBox="0 0 128 128" xml:space="preserve">
```

File: icons/googlecloud/googlecloud-plain-wordmark.svg [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```
Comment (amacado) [inline icons/googlecloud/googlecloud-plain-wordmark.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
	 viewBox="0 0 128 128" xml:space="preserve">
```

File: icons/googlecloud/googlecloud-original-wordmark.svg [hunk 3]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```
Comment (amacado) [inline icons/googlecloud/googlecloud-original-wordmark.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 
	 viewBox="0 0 128 128" xml:space="preserve">
```

File: icons/googlecloud/googlecloud-original.svg [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
```
Comment (amacado) [inline icons/googlecloud/googlecloud-original.svg:3]:
```suggestion
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" 
	 viewBox="0 0 128 128" xml:space="preserve">
```

**Review by amacado — CHANGES_REQUESTED** [review]
@EnisMulic Thanks for your contribution!

In the font versions (plain and workmark) of the svg you can see "cutting lines" which does not look like they are intended. 

image

Here are the `bot:peek` results as font preview:

new_icons
screenshot_0
screenshot_1

**Comment by Thomas-Boi** [comment]
Hello there,

Here are the icon build results:
new_icons

screenshot_0

screenshot_1

While the cut off lines have been fixed, I couldn't help but notice that the `plain-wordmark` icon looks like it's shifted upwards and it's different from before. I think it looks better before so is this change intentional? 

Besides that, everything looks great. If the change above is intentional, I can accept your PR. However, I think it would better centered.

**Comment by EnisMulic** [comment]
When I took a closer look the previous icons looked terrible when you would zoom in closer as the parts that it was made out of obviously looked detached and not aligned, so I replaces everything entirely. As for the shift, It's caused by setting the viewBox to 0 0 128 128 so I left it like that as I didn't know how to fix that and the guidelines say that should be the specified settings.

**Review by amacado — CHANGES_REQUESTED** [review]
@EnisMulic thanks for updating. The `0 0 128 128` requirement for the icons was developed to have all icons in the same view box which helps when making global changes. 
The off-centering does not look good and should be fixed before adding this icon to to project.

**Review by amacado — CHANGES_REQUESTED** [review]
Still a slight left/right margin difference (for original and plain), but much better!

image

**Comment by amacado** [comment]
@EnisMulic could you please request a review once you're done? So we don't have to check every commit.. :)

**Review by amacado — APPROVED** [review]
📦 and will be in the next release. Thanks @EnisMulic for the work! :) well done.

**Pull request merged**